package.json

```
@@ -0,0 +1,77 @@
+{
+  "name": "mdast-util-newline-to-break",
+  "version": "0.0.0",
+  "description": "mdast utility to support hard breaks without needing spaces or escapes",
+  "license": "MIT",
+  "keywords": [
+    "mdast",
+    "markdown",
+    "break",
+    "newline",
+    "linefeed"
+  ],
+  "repository": "remarkjs/remark-breaks",
+  "bugs": "https://github.com/syntax-tree/mdast-util-newline-to-break/issues",
+  "funding": {
+    "type": "opencollective",
+    "url": "https://opencollective.com/unified"
+  },
+  "author": "Titus Wormer <tituswormer@gmail.com> (https://wooorm.com)",
+  "contributors": [
+    "Remco Haszig <remcohaszing@gmail.com>",
+    "Titus Wormer <tituswormer@gmail.com> (https://wooorm.com)"
+  ],
+  "sideEffects": false,
+  "type": "module",
+  "exports": "./index.js",
+  "files": [
+    "lib",
```

```suggestion
    "lib/",
```

Explicit that it is a folder